src/coreComponents/physicsSolvers/solidMechanics/kernels/FixedStressThermoPoromechanics_impl.hpp

```diff
@@ -172,7 +179,32 @@ quadraturePointKernel( localIndex const k,
                                                                          stack.uhat_local,
                                                                          reinterpret_cast< real64 (&)[numNodesPerElem][3] >(stack.localResidual),
                                                                          -stabilizationScaling );
-  stiffness.template upperBTDB< numNodesPerElem >( dNdX, -detJxW, stack.localJacobian );
+  stiffness.template BTDB< numNodesPerElem >( dNdX, -detJxW, stack.localJacobian );
```

Comment:
@castelletto1  @joshua-white @CusiniM somehow this way does not work well for this specific case https://github.com/GEOS-DEV/GEOS/blob/develop/inputFiles/poromechanics/PoroElastic_hybridHexPrism_co2_sequential_3d.xml
any idea why?

Reply:
@castelletto1 is this way legal https://github.com/GEOS-DEV/GEOS/blob/15f8d14e65c3f3e340bbf407b084f1585725f14c/src/coreComponents/physicsSolvers/solidMechanics/kernels/FixedStressThermoPoromechanics_impl.hpp#L182 ?

Reply:
the porosity derivative does not seem to work properly, see initialization step for sequential run below:
```
  Iteration  1: linearElasticity
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  0
        ( Rsolid ) = ( 1.69e+01 )        ( R ) = ( 1.69e+01 )
        Last LinSolve(iter,res) = (   1, 2.88e-15 )
        linearElasticity: Global solution scaling factor = 1
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  1
        ( Rsolid ) = ( 1.63e-02 )        ( R ) = ( 1.63e-02 )
        Last LinSolve(iter,res) = (   1, 6.88e-15 )
        linearElasticity: Global solution scaling factor = 1
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  2
        ( Rsolid ) = ( 1.96e-05 )        ( R ) = ( 1.96e-05 )
        Last LinSolve(iter,res) = (   1, 7.26e-15 )
        linearElasticity: Global solution scaling factor = 1
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  3
        ( Rsolid ) = ( 2.42e-08 )        ( R ) = ( 2.42e-08 )

```
while here is the fully coupled initialization, converges in 1 iteration, no problem:
```
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  0
        ( Rflow ) = ( 0.00e+00 )        ( Rsolid ) = ( 1.69e+01 )        ( Rwell ) = ( 0.00e+00 )        ( R ) = ( 1.69e+01 )
        twoPhaseFlow: Max pressure change: 0.000 Pa (before scaling)
        twoPhaseFlow: Max component density change: 0.000 kg/m3 (before scaling)
        reservoirSystem: Global solution scaling factor = 1
        multiphasePoroelasticity: Max phase volume fraction change: 0.00
    Attempt:  0, ConfigurationIter:  0, NewtonIter:  1
        ( Rflow ) = ( 0.00e+00 )        ( Rsolid ) = ( 6.62e-14 )        ( Rwell ) = ( 0.00e+00 )        ( R ) = ( 6.62e-14 )
        Last LinSolve(iter,res) = (   1, 2.85e-15 )

```